harvard-lts/drs-import-management-service
# Update requirements.txt

need a new copy of this

**Update requirements.txt**
* * *


# What does this Pull Request do?
Update requirements library for install

# How should this be tested?

This is from local, should be Abel to build local, but best test is merge and let Jenkins rebuild and send to dev.

A description of what steps someone could take to:
* Reproduce the problem you are fixing (if applicable)
* Test that the Pull Request does what is intended.
* Please be as detailed as possible.
* Good testing instructions help get your PR completed faster.

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests?
- integration tests?

# Interested parties
Tag (@ mention) interested parties
